apps/veil/tsconfig.json

```diff
@@ -23,12 +23,14 @@
   },
   "extends": ["@tsconfig/strictest/tsconfig.json", "@tsconfig/vite-react/tsconfig.json"],
   "include": [
+    "app/**/*.ts",
+    "app/**/*.tsx",
+    "global.d.ts",
     "next-env.d.ts",
-    "**/*.ts",
-    "**/*.tsx",
```

Comment:
Including _all_ typescript files via `**/*.ts` was causing git-ignored paths to be considered, yuck. Fixed.